persistence/logs/log.log

```diff
@@ -0,0 +1,21 @@
+2021-01-19 17:07:44 [main] DEBUG springframework.jdbc.datasource.init.ScriptUtils - 1 returned as update count for SQL: INSERT into gift_certificate values (2, 'qqq','qqq', 12, 12, '2021-01-18T13:39:01.322', '2021-01-18T13:39:01.322')
+2021-01-19 17:07:44 [main] DEBUG springframework.jdbc.datasource.init.ScriptUtils - Executed SQL script from class path resource [createDatabase.sql] in 261 ms.
+2021-01-19 17:07:44 [main] DEBUG org.springframework.jdbc.core.JdbcTemplate - Executing SQL query [SELECT certificateId, name, description, price, duration, createDate, lastUpdateDate FROM gift_certificate]
+2021-01-19 17:07:44 [main] DEBUG org.springframework.jdbc.datasource.DataSourceUtils - Fetching JDBC Connection from DataSource
+2021-01-19 17:07:44 [main] DEBUG org.springframework.jdbc.datasource.SimpleDriverDataSource - Creating new JDBC Driver Connection to [jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false]
+2021-01-19 17:07:44 [main] INFO  springframework.jdbc.datasource.embedded.EmbeddedDatabaseFactory - Shutting down embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false'
+2021-01-19 17:07:44 [main] DEBUG org.springframework.jdbc.datasource.SimpleDriverDataSource - Creating new JDBC Driver Connection to [jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false]
+2021-01-19 17:07:44 [main] INFO  springframework.jdbc.datasource.embedded.EmbeddedDatabaseFactory - Starting embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false', username='sa'
```

Comment:
I would prefer not to commit this to git

Reply:
> I would prefer not to commit this to git

Thank you, fixed